marketplace_standard_app_api/models/transformation.py

```diff
@@ -67,3 +72,14 @@ class TransformationStateResponse(BaseModel):
 
 class TransformationListResponse(BaseModel):
     items: List[TransformationModel]
+
+
+class RegisteredModels(BaseModel):
+    message: Optional[str] = None
+    registered_models: List[ModelName]
+
+
+class NewModel(BaseModel):
+    parameters: dict
+
+    modelname: NewTransformationStates = TransformationState.CREATED
```

Comment:
Why do you need a new model? Can yo unot use the `NewTransformationModel`? And why is the `modelname` a state?